### PR TITLE
Haile orgpenalty

### PR DESCRIPTION
**Pull request by hailemof**

### CODE REVIEW

**Comment by hailemof** [comment]
Please Review and Merge

File: src/main/java/etgov/mof/pfmrt/conf/dao/MasterTransactionRepository.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+     
+	    @Query("SELECT z FROM MasterTransaction z WHERE CONCAT(z.id, ' ', z.createdBy) LIKE %?1%")
+	    public List<MasterTransaction> findTransactionByUsername(String  orgtrans);
+	    
+	    
+	    @Query("from MasterTransaction as m where m.user.id =:userid")
+		
+		public List<MasterTransaction> findTransactionByOrg(@Param("userid") Long id);
+	   
+		//public List<MasterTransaction> findByNameContainingAndUser(String docname,User user);
```
Comment (Wessi) [inline src/main/java/etgov/mof/pfmrt/conf/dao/MasterTransactionRepository.java:30]: Remove unnecessary lines of code and add comments.

File: src/main/java/etgov/mof/pfmrt/conf/model/OrgPenalty.java [hunk 2]
```diff
@@ -1,6 +1,6 @@
 package etgov.mof.pfmrt.conf.model;
 
-import java.sql.Date;
```
Comment (yosephtesfaye) [inline src/main/java/etgov/mof/pfmrt/conf/model/OrgPenalty.java:3]: should be java.util.Date;

Comment (hailemof) [reply src/main/java/etgov/mof/pfmrt/conf/model/OrgPenalty.java:3]: i accepted the remarks

File: src/main/java/etgov/mof/pfmrt/conf/dao/MasterTransactionRepository.java (hunk 1 above)
Comment (hailemof) [reply src/main/java/etgov/mof/pfmrt/conf/dao/MasterTransactionRepository.java:30]: noted with thanks

**Review by Wessi — APPROVED** [review]
All is Okay

**Comment by yosephtesfaye** [comment]
Done

**Review by yosephtesfaye — APPROVED** [review]
Done

**Pull request merged**